dashboard/app/views/pd/regional_partner_contact_mailer/_form_body.haml

```diff
@@ -0,0 +1,47 @@
+%p
+  %b Name:
+  - if @form[:title]
+    = "#{@form[:title]}" + ' '
+  = "#{@form[:first_name]}" + ' ' + "#{@form[:last_name]}"
+  %br
+  %b Email:
+  = "#{@form[:email]}"
+  %br
+  %b Job Role:
+  = "#{@form[:role]}"
+  %br
+  %b Job Title:
+  = "#{@form[:job_title]}"
+
+%p
+  %b School State:
+  = "#{@form[:school_state]}"
+  %br
+  %b School District:
+  - if @form[:school_district]
```

Comment:
I personally like the ternary operator for these situations because it's shorter, but this is also readable and totally fine. Ternary example: `= if @form[:school_district] ? "#{@form[:school_district]}" : "None"`

Reply:
Changing to ternary

Reply:
Left program field if/else however, because ternary was too long for one line